boteval/templates/admin/mturk/HIT.html

```diff
@@ -19,9 +19,11 @@ <h3>Assignments</h3>
       <div role="alert" class="alert {% if data['Assignments'] %}  alert-success {% else %}  alert-danger {% endif %}">
         Found {{ data['Assignments'] | length }} Assignments </div>
       <ol class="list-group">
+        {% set index = 0 %}
         {% for asgn in data['Assignments'] %}
         <li class="list-group-item">
-          {{ render_assignment(asgn, qtypes, crowd_name=meta['crowd_name'])}}
+          {{ render_assignment(asgn, qtypes, base_pay, pay_per_hour, bonus_pay[index], crowd_name=meta['crowd_name'])}}
```

Comment:
just a question: I see where asgn is coming from, where are the other parameters defined and passed over here? e.g. base_pay, pay_per_hour, bonus_pay?

Reply:
They're defined in the mturk.py file, in the get_HIT() method!